tests/entrypoints/openai/test_chat.py

```diff
@@ -7,11 +7,11 @@
 import openai  # use the official client for correctness check
 import pytest
 import torch
-# downloading lora to test lora requests
-from huggingface_hub import snapshot_download
 from openai import BadRequestError
 
 from ...utils import RemoteOpenAIServer
+from .test_completion import zephyr_lora_added_tokens_files  # noqa: F401
```

Comment:
Are these `noqa` statements necessary?

Reply:
They seem to be yes, it thinks they are unused but they are required by the fixtures.